incubator/binding-openapi/src/main/java/io/aklivity/zilla/runtime/binding/openapi/config/OpenapiOptionsConfig.java

```diff
@@ -41,11 +42,13 @@ public static <T> OpenpaiOptionsConfigBuilder<T> builder(
     }
 
     public OpenapiOptionsConfig(
```

Comment:
```suggestion
    OpenapiOptionsConfig(
```
This constructor should be package private as we have a builder.